src/sc/sidechain.h

```diff
@@ -90,19 +101,57 @@ class ScCoinsViewCache
 
 class ScMgr
 {
-  private:
+public:
+    enum persistencePolicy {
+        STUB = 0, //utility for UTs
+        PERSIST,
+    };
+
+private:
+    class persistenceLayer {
```

Comment:
Pls use an Upercase initial in the name of the class. The same applies also for the other ones below.

Reply:
Agree. Change performed